library/build.gradle

```diff
@@ -20,6 +20,15 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/java'
+        test.java.srcDirs += 'src/test/kotlin'
+    }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
`Log.x` causes isses at test runtime. This causes such classes to return logical defaults, rather than throw exceptions.

Reply:
Good to know, that is something I was unaware of! Thanks for sharing :+1: